------------------------------------------------------------
revno: 12484
committer: Janet Anderson <jba@aps.anl.gov>
branch nick: 3.14
timestamp: Mon 2014-04-14 15:00:37 -0500
message:
  mkmf.pl needs -I prefix on include directories.
diff:
=== modified file 'configure/CONFIG_COMMON'
--- configure/CONFIG_COMMON	2013-07-10 19:13:31 +0000
+++ configure/CONFIG_COMMON	2014-04-14 20:00:37 +0000
@@ -323,8 +323,7 @@
 HDEPENDS = YES
 HDEPENDS_METHOD = CMD
 
-HDEPENDS_INCLUDES = $(subst -I,,$(INCLUDES))
-HDEPENDSFLAGS = -m $*$(DEP) $(HDEPENDS_INCLUDES) $@ $<
+HDEPENDSFLAGS = -m $*$(DEP) $(INCLUDES) $@ $<
 HDEPENDSCMD = -$(MKMF) $(HDEPENDS_FLAGS) $(HDEPENDSFLAGS)
 
 HDEPENDS_CMD_NO = $(ECHO) ""
